source/_components/google_cloud_tts.markdown

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: "Google Cloud TTS"
+description: "Google Cloud text-to-speach with Home Assistant."
+date: 2019-05-02 12:08
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: google_cloud_tts.png
+ha_category: Text-to-speech
+ha_release: "0.92.3"
```

Comment:
We don't target patch version, please change it to 0.93

Reply:
Fixed in 68f0c6a65e31a4967d4f36eacfe767aad55a4eb3